spec/routes/web/project/billing_spec.rb

```diff
@@ -433,6 +441,23 @@ def billing_record(begin_time, end_time)
         expect(text).to include("Tax ID: 123123123")
       end
 
+      it "show persisted invoice PDF from blob storage" do
+        expect(Stripe::Customer).to receive(:retrieve).with("cs_1234567890").and_return({"name" => "ACME Inc.", "address" => {"country" => "US"}, "metadata" => {"company_name" => "Foo Companye Name", "tax_id" => "123123123"}}).at_least(:once)
+        bi = billing_record(Time.parse("2023-06-01"), Time.parse("2023-07-01"))
```

Comment:
```suggestion
        bi = billing_record(Time.utc(2023, 6), Time.utc(2023, 7))
```

This is simpler, plus it's better to do things in UTC instead of local time (`Time.parse` uses local time).

Reply:
Updated some tests at d0cdf2ee5a22e7aa25b918606f128342fd0c3cda